.reuse/dep5

```diff
@@ -27,3 +27,7 @@ Disclaimer: The code in this project may include calls to APIs (“API Calls”)
 Files: *
 Copyright:  2020-2021 SAP SE or an SAP affiliate company and project "Sailor" contributors
 License: Apache-2.0
+
+Files: docs/_templates/apidoc/module.rst_t, docs/_templates/apidoc/package.rst_t
+Copyright:  2007-2021 by the Sphinx team.
```

Comment:
If I recall correctly, you mentioned that you modified their original code. In this case, that should also be mentioned here:

```suggestion
Copyright:  2007-2021 by the Sphinx team. Modifications 2021 SAP SE or an SAP affiliate company and project "Sailor" contributors
```